src/pages/myPage/MyNeighborhoodVerificationPage.tsx

```
@@ -221,13 +32,39 @@ const MyNeighborhoodVerificationPage = () => {
         }
     }, [errorMessage]);
 
+    // 카카오맵 SDK 로드 대기
+    useEffect(() => {
+        // 이미 로드되어 있으면 바로
+        if ((window as any).kakao?.maps?.services) {
```

ESLint

[error] Unexpected any. Specify a different type.

(`@typescript-eslint/no-explicit-any`)

---

[error] Error: Calling setState synchronously within an effect can trigger cascading renders

Effects are intended to synchronize state between React and external systems such as manually updating the DOM, state management libraries, or other platform APIs. In general, the body of an effect should do one or both of the following:
* Update external systems with the latest state from React.
* Subscribe for updates from some external system, calling setState in a callback function when external state changes.

Calling setState synchronously within an effect body causes cascading renders that can hurt performance, and is not recommended. (https://react.dev/learn/you-might-not-need-an-effect).

/home/jailuser/git/src/pages/myPage/MyNeighborhoodVerificationPage.tsx:39:13
  37 |         // 이미 로드되어 있으면 바로
  38 |         if ((window as any).kakao?.maps?.services) {
> 39 |             setIsKakaoReady(true);
     |             ^^^^^^^^^^^^^^^ Avoid calling setState() directly within an effect
  40 |             return;
  41 |         }
  42 |

(react-hooks/set-state-in-effect)